modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java

```diff
@@ -664,6 +665,12 @@ public Map<String, Object> getEditorContext() throws Exception {
 			).put(
 				"styleBookEnabled",
 				() -> {
+					if (FeatureFlagManagerUtil.isEnabled(
+							themeDisplay.getCompanyId(), "LPD-30204")) {
+
+						return true;
+					}
+
 					Layout layout = themeDisplay.getLayout();
```

Comment:
Other methods in this that fill the attributes in this `HashMapBuilder` use `Layout`, `Theme` and other info from the themeDisplay to build their values.